src/services/publisher_service/helpers.rs

```diff
@@ -90,6 +89,7 @@ pub struct NotificationToProcess {
     pub subscriber_sym_key: String,
     #[sqlx(try_from = "String")]
     pub subscriber_topic: Topic,
+    pub subscriber_notification: Uuid,
```

Comment:
Non blocking: Maybe `subscriber_notification_id` is more self-describing here.

Reply:
Hmm, I know where you are coming from.

The lack of `_id` is a pattern all over the place which I think stemmed from the existence of `project_id` which is different from `project.id`, so I ended up overloading `project` which could be `Project` or `Uuid`, and applied the same pattern to all types both variable and column names. Can pretty easily see if it is the object, or the ID with the type and IDE hints. Not ideal, but just going to keep it as-is for sake of consistency and discussion time :)